### ISSUE TITLE
rearrange directory structure

### ISSUE THREAD

**Issue by nbassler** [nbassler/TRiPddd#11]
```
src/SH12A/ddd
src/SH12A/spc
src/TOPAS/ddd
src/TOPAS/spc

base/00_SH12A_generic/README.md       # readme contains a short description + link to zenodo repo?
base/01_TOPAS_generic/README.md       #
base/02_SH12A_DCPT/README.md         # with some beam model of dcpt
base/03_SH12A_CCBg1/README.md         # with some beam model of the CCB Gantry-1
...
```
on zenodo:
```
00_SH12A_generic/README.md 

00_SH12A_generic/ddd/1H
00_SH12A_generic/ddd/4He
00_SH12A_generic/ddd/12C

00_SH12A_generic/spc/1H
00_SH12A_generic/spc/4He
00_SH12A_generic/spc/12C

00_SH12A_generic/dedx/20130806.dedx
00_SH12A_generic/hlut/..
00_SH12A_generic/sis/..
```
?

**Comment by grzanka** [comment]
On Zenodo I'd upload an .tar.gz archives, i.e. `00_SH12A_generic.tar.gz` accompanied with description  `00_SH12A_generic.md`. This is due to the fact that downloading multiple (small) files is not easy with zenodo

**Comment by grzanka** [comment]
What about renaming `base` directory to `data` ?

**Comment by nbassler** [comment]
> What about renaming `base` directory to `data` ?

fine with me.

**Comment by grzanka** [comment]
I was a bit inspired by the project structure typically used in ML, like this one: https://drivendata.github.io/cookiecutter-data-science/

**Comment by nbassler** [comment]
closed with PR #12
